plugin/output/loki/loki.go

```diff
@@ -0,0 +1,425 @@
+package loki
+
+import (
+	"bytes"
+	"context"
+	"crypto/tls"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+
+	"github.com/ozontech/file.d/cfg"
+	"github.com/ozontech/file.d/fd"
+	"github.com/ozontech/file.d/metric"
+	"github.com/ozontech/file.d/pipeline"
+
+	insaneJSON "github.com/ozontech/insane-json"
+	"github.com/prometheus/client_golang/prometheus"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+/*{ introduction
+It sends the logs batches to Loki using HTTP API.
+}*/
+
+const (
+	outPluginType = "loki"
+)
+
+type data struct {
+	outBuf []byte
+}
+
+type Lable struct {
+	Label string `json:"label" required:"true"`
+	Value string `json:"value" required:"true"`
+}
+
+// ! config-params
+// ^ config-params
+type Config struct {
+	// > @3@4@5@6
+	// >
+	// > A full URI address of Loki
+	// >
+	// > Example address
+	// >
+	// > http://127.0.0.1:3100 or https://loki:3100
+	Address string `json:"address" required:"true"` // *
+
+	// > @3@4@5@6
+	// >
+	// > Array of labels to send logs
+	// >
+	// > Example labels
+	// >
+	// > label=value
+	Labels []Lable `json:"labels" required:"true"`
+
+	// > @3@4@5@6
+	// >
+	// > Message field from log to be mapped to loki
+	// >
+	// > Example
+	// >
+	// > message
+	MessageField string `json:"message_field" required:"true"`
+
+	// > @3@4@5@6
+	// >
+	// > Timestamp field from log to be mapped to loki
+	// >
+	// > Example
+	// >
+	// > timestamp
+	TimestampField string `json:"timestamp_field" required:"true"`
+
+	// > @3@4@5@6
+	// >
+	// > Authorization enabled, if true set OrgID
+	AuthEnabled bool `json:"auth_enabled" default:"false"` // *
+
+	// > @3@4@5@6
+	// >
+	// > Authorization enabled, if set true set OrgID
+	// >
+	// > Example organization id
+	// >
+	// > example-org
+	OrgID string `json:"org_id"` // *
```

Comment:
I propose call it tenant_id

https://vector.dev/docs/reference/configuration/sinks/loki/#tenant_id
https://grafana.com/docs/loki/latest/send-data/promtail/configuration/#clients
https://grafana.com/docs/loki/latest/send-data/promtail/configuration/#clients